Vagrantfile

```diff
@@ -21,37 +21,55 @@ else
   abort
 end
 
+vbox_version = `VBoxManage --version`.strip.match(/^(\d+\.\d+\.\d+)/)[1]
+
 vm_name = "tracee-#{arch}-vm"
+vm_user = "vagrant"
+vm_synced_folder = "/#{vm_user}"
+
+provider_settings = lambda do |prov, name|
+  if name == "virtualbox"
+    prov.name = vm_name
+    prov.cpus = "16"
+    prov.memory = "16318"
+    prov.gui = false
+  elsif name == "qemu"
+    prov.cpu = "16"
+    prov.memory = "16G"
+  end
+end
 
 Vagrant.configure("2") do |config|
+  # disable automatic folder mounting, as it is done manually in the last provision stage
+  config.vm.synced_folder ".", "#{vm_synced_folder}", auto_mount: false
+
   config.vm.network "forwarded_port", guest: 9090, host: 9090
   config.vm.network "forwarded_port", guest: 3366, host: 3366
   config.vm.network "forwarded_port", guest: 3000, host: 3000
-  case arch
-  when "amd64"
-    # config.vm.box = "ubuntu/focal64"     # Ubuntu 20.04 Focal Fossa (non CO-RE)
-    # config.vm.box = "ubuntu/hirsute64"   # Ubuntu 21.04 Hirsute Hippo (CO-RE)
-    # config.vm.box = "ubuntu/impish64"    # Ubuntu 21.10 Impish Indri (CO-RE)
-    config.vm.box = "ubuntu/jammy64"       # Ubuntu 22.04 Jammy Jellyfish (CO-RE)
-  when "arm64"
-    config.vm.box = "bento/ubuntu-22.04-arm64"
-  end
+
+  # virtualbox, parallels, vmware_desktop, qemu, libvirt
+  config.vm.box = "bento/ubuntu-22.04"
 
   case host_os
   when "Linux"
     config.vm.provider "virtualbox" do |vb|
-      vb.name = vm_name
-      vb.cpus = "8"
-      vb.memory = "4096"
-      vb.gui = false
+      provider_settings.call(vb, "virtualbox")
     end
   when "Darwin"
-    config.vm.provider "parallels" do |prl|
-      prl.name = vm_name
+    case arch
+    when "amd64"
+      config.vm.provider "virtualbox" do |vb|
+        provider_settings.call(vb, "virtualbox")
+      end
     end
+    when "arm64"
+      config.vm.provider "qemu" do |qe|
+        provider_settings.call(qe, "qemu")
```

Comment:
If the box works on Linux/Darwin amd64/arm64, we should update documentation replacing mention to parallels.